x-pack/plugins/cases/public/components/add_comment/index.test.tsx

```diff
@@ -115,6 +114,19 @@ describe('AddComment ', () => {
     ).toBeTruthy();
   });
 
+  it('should hide the component when the user does not have crud permissions', () => {
+    usePostCommentMock.mockImplementation(() => ({ ...defaultPostComment, isLoading: true }));
+    const wrapper = mount(
+      <TestProviders>
+        <Router history={mockHistory}>
+          <AddComment {...{ ...addCommentProps, userCanCrud: false }} />
+        </Router>
+      </TestProviders>
+    );
+
+    expect(wrapper.find(`[data-test-subj="sadd-comment"]`).exists()).toBeFalsy();
```

Comment:
nit: What `s` means (first letter of `sadd-comment`)?

Reply:
Nice catch! that data-test-subj definitely wouldn't have existed 😆  should be `add-comment`